### ISSUE TITLE
rollbar.configure reinstruments console log and hijacking console.log stack trace again

### ISSUE THREAD

**Issue by mpospelov** [rollbar/rollbar.js#655]
Hello everyone. I have the issue similar to #609. Here is my use case:
1. User visits webapp - `new rollbar` is called - no hijacking to console.log
2. User logins - `rollbar.configure({ payload: {user_id: <...>} })` is called and autoinstumenter hijacks to console.log

So is it an expected behaviour?
I found a workaround to pass to `enabled` during configure same as init but it looks awkward.

here is my code, to make it more error prone I made rollbar instance private and call all methods only in this file
```
import RollbarClass from 'rollbar';

const enabled = ['production', 'staging', 'preproduction'].includes(process.env.SERVER_ENV);

const configureOptions = {
  enabled,
  accessToken: global.__SERVER__
    ? process.env.ROLLBAR_SERVER_ACCESS_TOKEN
    : process.env.ROLLBAR_CLIENT_ACCESS_TOKEN,
  captureUncaught: true,
};

const Rollbar = new RollbarClass({
  ...configureOptions,
  payload: {
    environment: process.env.SERVER_ENV,
    isServer: global.__SERVER__,
    client: {
      javascript: {
        code_version: process.env.GIT_SHA,
        guess_uncaught_frames: true,
      },
    },
  },
});

export function reportError(error, info) {
  Rollbar.error(error, info);
}

export function configureReporter(payload) {
  Rollbar.configure({ ...configureOptions, payload });
}
```

**Comment by shakiba** [comment]
@mpospelov I can't reproduce this, can you please share an example that I run to reproduce it.

**Comment by mpospelov** [comment]
@shakiba Hi. Please checkout this `create-react-app` repo
`https://github.com/mpospelov/rollbar-configure-issue`
1. Start it with `yarn start`
2. checkout console.log
https://github.com/mpospelov/rollbar-configure-issue/blob/master/src/App.js#L8-L10
![image](https://user-images.githubusercontent.com/1548210/44617933-d9e2eb00-a874-11e8-884d-585a28923961.png)


According to settings rollbar should be disabled, and should not instrument `console.log` but as you can see `console.log(2)` stack trace is broken. See config for rollbar
https://github.com/mpospelov/rollbar-configure-issue/blob/master/src/rollbar.js

**Comment by shakiba** [comment]
@mpospelov Thanks for the demo! I can see inconsistency here and I figured out where the issue, however I'm still not sure what the best solution is.

For now I recommend adding `autoInstrument: false` or `autoInstrument: { log: false }` to config object.

**Comment by mpospelov** [comment]
@shakiba I solved my issue by always passing enabled flag with config, checkout the code in the question. 

In demo if you pass Rollbar.configure({ enabled: false, payload: {...}}), rollbar wont autoinstrument console.

**Comment by shakiba** [comment]
Right, but that will disable everything (no data will be sent to the server anymore).

**Comment by mpospelov** [comment]
I do not care about log instrumentation in production so it's a development issue, I have rollbar disabled then I do `Rollbar.configure` and it starts instrumenting console like it's enabled, but it shouldn't, that's why I always passing initial `configureOptions`

**Comment by shakiba** [comment]
Thanks for info!  I made a PR #660, I will merge it and make a new release soon.

**Comment by shakiba** [comment]
I released v2.4.5 with bug fix for this issue.
